src/custom/components/MenuDropdown/index.tsx

```diff
@@ -9,16 +9,23 @@ interface MenuProps {
 export function Menu({ title, children }: MenuProps) {
   const [showMenu, setShowMenu] = useState(false)
   const handleOnClick = () => setShowMenu(true)
-  const handleMouseEnter = () => setShowMenu(true)
-  const handleMouseLeave = () => setShowMenu(false)
+  // const handleMouseEnter = () => setShowMenu(true)
+  // const handleMouseLeave = () => setShowMenu(false)
```

Comment:
Not sure why you are disabling the hover events here, but now I cannot close the menu 😄 

Anyway, I trust this will be changed in a later PR, so no worries